client/app/controllers/collection/add.js

```diff
@@ -83,43 +11,125 @@ export default Ember.Controller.extend({
     }),
 
     widgets: [],
+    actions: [],
```

Comment:
as with the widgets arr, actions arr allows easier manipulation of the actions associated with the form.